File: C/PRAK402-2210817310009-MarioFrancaWijaya-.c
#include <stdio.h>

int main(){
    int i,j, angka;
    scanf("%d", &angka);

    for(i=1; i<=angka;i++){
        if(i % 2==0){
        }else{
            printf("%d ", i);
        }
    }
    printf("\n");
    for(j=angka;j>=1;j--){
        if(j % 2 == 0){
            printf("%d ", j);
        }
    }
    
    return 0;
}

File: Python/PRAK402-2210817310009-Mario Franca Wijaya.py
angka = int(input())
for i in range(1, angka):
    if i % 2 != 0 :
        print(i, end = ' ')
print('\n')
while angka > 0:
    if angka % 2 == 0:
        print(angka, end = ' ')
    angka -= 1
